gcp/appengine/frontend_handlers.py

```diff
@@ -297,39 +297,41 @@ def bug_to_response(bug, detailed=True):
   return response
 
 
+def calculate_severity_details(
+    severity: dict) -> tuple[float | None, str | None]:
+  """Calculate score and rating of severity"""
```

Comment:
I cherry-picked this commit from my other branch which adds[ link to cvss calculator.](https://github.com/google/osv.dev/issues/2254)
FYI - Once this PR was merged, I'll resolve the potential conflicts and open a seperate PR for the mentioned issue.